cms/djangoapps/contentstore/course_group_config.py

```diff
@@ -18,11 +18,11 @@
 RANDOM_SCHEME = "random"
 COHORT_SCHEME = "cohort"
 
-# Note: the following content group configuration strings are not
-# translated since they are not visible to users.
-CONTENT_GROUP_CONFIGURATION_DESCRIPTION = 'The groups in this configuration can be mapped to cohort groups in the LMS.'
+CONTENT_GROUP_CONFIGURATION_DESCRIPTION = _(
+    'The groups in this configuration can be mapped to cohort groups in the LMS.'
```

Comment:
Sure, I can make that change. But this particular string isn't user-visible (I just marked it for translation in case that changes, since the name now IS user-visible).

I am also going to change "LMS" to "Instructor Dashboard".